README.md

```diff
@@ -12,6 +12,11 @@ To install the requirements first run the command
 ```
 sudo pip install -r requirements.txt
 ```
+For Windows users, the word "sudo" should not be used. Hence, you should run the command:
```

Comment:
no you should make the change locally yourself, then `git add .`  then `git commit --amend` and then `git push -f` this will ammend your changes on your previous commit